### PR TITLE
Notification service v1

### LINKED ISSUE #28: Make notification service compatible with Insights Notification service backend

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Incomplete but working version of the notification service.

- Config files have been cleaned up. Removed values are expected to be set in deployment environment.
- Differ produces 1 notification per cluster with important and critical issues.
- Updated constants for notification message (fields expected by the email template and such)
- Use the timestamp of the selected reports as timestamp for the notified issues
- Notify issues with total risk higher or equal to 3 instead of 2
- Make the notification message comply with the expected format (JSON string with escaped strings inside) 
- Fix some errors in the code (e.g Kafka producer was set and closed for each cluster), added some error control and basic logging.
- Added TODOs for code that is part of other tasks

Fixes #28 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update
- Refactor (refactoring code, removing useless files)

## Testing steps

Mostly manual testing while preparing the demo.

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: PoC-differ/differ.go [hunk 1]
```diff
@@ -179,54 +207,49 @@ func processClusters(ruleContent map[string]types.RuleContent, impacts types.Glo
...
 		}
 
-		log.Info().Msg(separator)
-		log.Info().Msg("Preparing Kafka producer")
-
-		notifier := setupNotificationProducer(kafkaConfig)
-		log.Info().Msg("Kafka producer ready")
-
-		waitForEnter()
+		if len(deserialized.Reports) != 0 {
```
Comment (tisnik) [inline PoC-differ/differ.go:214]: probably the opposite condition would be better, as "happy path" in Go points down, not to the left

**Review by tisnik — CHANGES_REQUESTED** [review]
LGTM in overall

File: PoC-differ/differ.go (hunk 1 above)
Comment (epapbak) [reply PoC-differ/differ.go]: Done! Thanks for the tip

**Review by tisnik — APPROVED** [review]
LGTM

**Pull request merged**